pkg/tsl/consts.go

```diff
@@ -31,6 +31,7 @@ const (
 	RegexOp      = "$regex"
 	NotRegexOp   = "$nregex"
 	LikeOp       = "$like"
+	ILikeOp      = "$ilike"
 	NotLikeOp    = "$nlike"
```

Comment:
can we drop the `NotLikeOp` here and in the walkiers ( for example here https://github.com/yaacov/tree-search-language/blob/master/pkg/walkers/sql/walk.go#L142) ?

Can we generate it now ?

Reply:
Yes; I think it can be dropped. Do you want that in this same pull request? I'd prefer a different one.

Reply:
Regarding your question about generating: I already executed the code generator, if that is what you are asking.

Reply:
> Regarding your question about generating: I already executed the code generator, if that is what you are asking.

No :-)
I asked if we can write a query that will generate it in the tree.

Reply:
No, the parser will never generate that kind of tree; it combines NotOp with Like or Ilike. See #13.